spec/actions/pod_push_with_error_handling.rb

```diff
@@ -19,6 +23,16 @@ def self.run(params)
           if output_str.include?("[!] Unable to accept duplicate entry for:")
             UI.error("⚠️ Duplicate entry detected. Skipping push.")
             return false
+          elsif output_str.include?("[!] Calling the GitHub commit API timed out.")
```

Comment:
I wonder if we should also retry upon an "Internal server error"... That also happens every now an then 😓

Reply:
Do you have a link or a log message?? I can add that in another PR for clarity

Reply:
I was getting `[!] An internal server error occurred. Please check for any known status issues at https://twitter.com/CocoaPods and try again later.` before